hack/local-up-cluster.sh

```diff
@@ -692,11 +692,11 @@ function wait_node_ready(){
   # check the nodes information after kubelet daemon start
   local nodes_stats="${KUBECTL} --kubeconfig '${CERT_DIR}/admin.kubeconfig' get nodes"
   local node_name=$HOSTNAME_OVERRIDE
-  local system_node_wait_time=60
+  local system_node_wait_time=150
```

Comment:
@dims should we make this configurable for users?

Reply:
@hasheddan +1

Reply:
ok, let me make it configurable.